WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -60,6 +57,11 @@
 
               <td class="name"><%= registration.name %></td>
               <td class="countryId"><%= registration.countryId %></td>
+              <td>
+                <span data-toggle="tooltip" data-placement="left" data-container="body" title="<%= registration.created_at %>">
+                  <%= registration.created_at.to_date %>
```

Comment:
Oh that's tricky. Bootstrap-table sorts alphabetically content of the 'td's, so in this case first difference it always stumble upon is the title which contains full registration date (even with the hour). Further more the format of dates in our case is perfect because it contains months as numbers and also the importance of parts of a date goes from left to right. Thus we can belong on the normal sorting and it will work just fine.